frontend/src/bundles/common/components/dropdown/components/dropdown.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable unicorn/prefer-ternary */
```

Comment:
Why we disable eslint there?

Reply:
when I use if operator, eslint suggests ternaty operator
but when I use ternary, there is conflict with eslint and prettier

Reply:
ok